platform/android/src/ch/opengis/qfield/QFieldActivity.java

```diff
@@ -159,4 +174,42 @@ private void checkPermissions()
             ActivityCompat.requestPermissions(QFieldActivity.this, permissions, 101);
         }
     }
+
+    private void checkAllFileAccess()
+    {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R && !Environment.isExternalStorageManager() && !sharedPreferences.getBoolean("DontAskAllFilesPermission", false)) {
+            AlertDialog.Builder builder = new AlertDialog.Builder(this);
+            builder.setTitle("All files access");
+            builder.setMessage(Html.fromHtml("QField needs all files access permission to be able to open local projects and datasets as well as supporting custom projection grids, fonts, and base maps. While not required, <i>allowing access is highly recommended</i>.<br><br>On the next screen, please select <b>QField</b> and select the <b>Allow access to all files</b> option.", Html.FROM_HTML_MODE_LEGACY));
```

Comment:
Should this come from the xml?

Reply:
Yes it should. For some reason it fails though. I must be missing a really basic thing here.

Reply:
Will fix in a follow up PR. Let's see if it uploads to store.